test/getFavicon.test.ts

```diff
@@ -0,0 +1,76 @@
+import { Response } from 'light-my-request'
+import sharp from 'sharp'
+import { describe, expect, it } from 'vitest'
+import createApp from '../src/server'
+
+const app = createApp()
+
+// Fetch from the API
+const mockFetch = (url: string): Promise<Response> => app.inject({ method: 'GET', url })
+
+// Fetch an array of URLs
+const fetchArray = (urls: string[], size: number | undefined = undefined) => Promise.all(urls.map(url =>
+  mockFetch(`/${encodeURIComponent(url)}${size ? `/${size}` : ''}`)
+))
+
+describe('getFavicon', async () => {
+  const has_favicon = [
+    'google.com',
```

Comment:
Feels like it might be nice to externalise this list. Maybe as a seperate JSON file perhaps?